src/apps/ocioview/ocioview/README.md

```diff
@@ -0,0 +1,48 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the OpenColorIO Project. -->
+
+ocioview (alpha)
+================
+
+**Work in progress**. ``ocioview`` is a visual editor for OCIO configs, written in 
+Python.
+
+The app currently consists of three main components; a viewer, a config editor, and a 
+transform and config inspector. Multiple viewers can be loaded in different tabs. The 
+config editor is a tabbed model/view interface for the current config. Models for 
+each config item type interface directly with the config in memory. The inspector 
+presents interfaces for inspecting processor curves, serialized config YAML, CTF and 
+shader code, and the OCIO log.
+
+The app's scene file is a config. This design allows dynamic interconnectivity between 
+config items, reducing risk of errors during config authoring. Undo/redo stack support 
+for most features is also implemented.
+
+These components are linked with 10 possible transform subscriptions. Each subscription 
+tracks the transform(s) for one config item, and each viewer can subscribe to any of 
+these transforms, providing fast visual feedback for transform editing.
+
+``ocioview`` being an alpha release means this app is functional, but still in development, so 
+may have some rough edges. Feedback and bug reports are appreciated.
+
```

Comment:
It doesn't work on Mac w/out Remi's fixes, right? Perhaps add:
Development has mostly been done on Windows. Better support for other platforms is forthcoming.